docs/source/hpxml_to_openstudio.rst

```diff
@@ -592,8 +592,23 @@ The following elements, some adopted from the `PVWatts model <https://pvwatts.nr
 - ``ArrayAzimuth``
 - ``ArrayTilt``
 - ``MaxPowerOutput``
-- ``InverterEfficiency``: Default is 0.96.
-- ``SystemLossesFraction``: Default is 0.14. System losses include soiling, shading, snow, mismatch, wiring, degradation, etc.
+
```

Comment:
It looks like the documentation needs to be updated for your most recent changes (appliances location, water heater location, hot water piping length).